plugins/MauticCrmBundle/Integration/CrmAbstractIntegration.php

```diff
@@ -550,10 +550,8 @@ protected function cleanPriorityFields($fieldsToUpdate, $objects = null)
      */
     protected function getSyncTimeframeDates(array $params)
     {
-        $fromDate = (isset($params['start'])) ? \DateTime::createFromFormat(\DateTime::ISO8601, $params['start'])->format('Y-m-d H:i:s')
-            : null;
-        $toDate = (isset($params['end'])) ? \DateTime::createFromFormat(\DateTime::ISO8601, $params['end'])->format('Y-m-d H:i:s')
-            : null;
+        $fromDate = (isset($params['start'])) ? \DateTime::createFromFormat(\DateTime::ATOM, $params['start'])->setTimezone(new \DateTimeZone('UTC'))->format('Y-m-d H:i:s') : null;
```

Comment:
Any reason why the date format was changed? Was it causing issues? I suggest one more improvement:
```suggestion
        $fromDate = (isset($params['start'])) ? \DateTime::createFromFormat(\DateTime::ATOM, $params['start'])->setTimezone(new \DateTimeZone('UTC'))->format(DateTimeHelper::FORMAT_DB) : null;
```

Reply:
createFromFormat take timezone from server. If we use `Europe/Paris` the command for get leads/companies take time diff from that timezone. But we have date times in DB  in UTC.  These is reason why we need force it to UTC

Reply:
I understand the change in the timezone. I was asking about the date format change which was not explained in the PR description :)

Reply:
I've changed it based on PHP Storm docs block:

```
    /**
     * This format is not compatible with ISO-8601, but is left this way for backward compatibility reasons.
     * Use DateTime::ATOM or DATE_ATOM for compatibility with ISO-8601 instead.
     * @since 7.2
     * @deprecated
     */
    public const ISO8601 = 'Y-m-d\TH:i:sO';
```

Reply:
I did not know about it. Thanks for this!